### PR TITLE
PaloAlto PAN-OS OnPrem playbooks and connector versions. Version - 2

### PR DESCRIPTION
**Pull request by ospeka**
Created OnPrem versions of playbooks and connector.
Change(s):

added BlockIP OnPrem playbook
added BlockURL OnPrem playbook
added GetSystemInfo OnPrem playbook
added GetThreatPCAP OnPrem playbook
added GetURLCategoryInfoOnPrem playbook
added PAN-OS Logic Apps OnPrem Custom connector

This PR is update for old PR - https://github.com/Azure/Azure-Sentinel/pull/6561

### CODE REVIEW

**Comment by v-prasadboke** [comment]
Hello @ospeka, Thanks for raising this PR. This PR will be investigated and will update you about the same before 22 December, 2023.

**Comment by v-prasadboke** [comment]
Hello @ospeka, can you share the working images of playbooks. Thanks.

**Comment by v-prasadboke** [comment]
Hello @ospeka, we are facing challenge while repackaging the solution due to this new connector. We are trying to resolve this issue and get back to you soonest.

**Comment by v-prasadboke** [comment]
Hello @ospeka some of the playbooks have missing metadata can you please add those to them.

**Comment by v-prasadboke** [comment]
Hello @ospeka please check on the playbook once and add on the required fields

**Comment by v-prasadboke** [comment]
Hello @ospeka, Can you please add the required missing fields

**Comment by v-prasadboke** [comment]
Hello @ospeka, Please proceed with the above requested changes

**Comment by v-atulyadav** [comment]
Hi @ospeka, please check above comments and act accordingly. Thanks

**Comment by v-prasadboke** [comment]
Hello @ospeka, Please work on the above requested changes

**Comment by ospeka** [comment]
@v-prasadboke  Hhello. I've added metadata to PAN-OS-GetURLCategoryInfo playbook

**Comment by v-prasadboke** [comment]
Thank you @ospeka for the updates, I'll check on it and update you by 25 January, 2024.

**Comment by v-prasadboke** [comment]
Hello @ospeka, Sorry for the inconvenience we are still working on it and facing some issue while repackaging the solution. Will get back to you with an update by 29 January, 2024.

File: Solutions/PaloAlto-PAN-OS/Playbooks/PaloAltoCustomConnectorOnPrem/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,2478 @@
...
+   "parameters":{
+      "PaloAlto_PAN-OS_Rest_API_CustomConnector_Name":{
+         "defaultValue":"PAN-OSRestApiCustomConnector",
+         "type":"String",
+         "metadata":{
+            "description":"Name of the PaloAlto REST API Connector"
+         }
+      },
+      "PaloAlto_PAN-OS_XML_API_CustomConnector_Name":{
+         "defaultValue":"PAN-OSXmlApiCustomConnector",
```
Comment (v-prasadboke) [inline Solutions/PaloAlto-PAN-OS/Playbooks/PaloAltoCustomConnectorOnPrem/azuredeploy.json:17]: This title is already taken by other connector in the Solution. Please check once

**Comment by v-prasadboke** [comment]
Screenshot (129)
 
Also there are some Hardcoded location in the playbook please take a look at them too

**Comment by v-prasadboke** [comment]
Screenshot (129)
 
Also there are some Hardcoded location in the playbook please take a look at them too

**Comment by v-prasadboke** [comment]
Hello @ospeka, Please take a look at the above mentioned errors

**Comment by v-prasadboke** [comment]
Hello @ospeka, Please provide some feedback on the above mentioned comments.

**Comment by v-prasadboke** [comment]
Hello @ospeka, we are waiting for your feedback

**Comment by v-prasadboke** [comment]
Hello @ospeka, we are waiting for your feedback

**Comment by v-prasadboke** [comment]
We wanted to check on the status of PR #9638. PR is pending for more than 30 days. Please let us know if you need any assistance to review this PR. Per our standard operating procedures if no response is received in the next 7 business days, we will close this PR. Thank you for your cooperation.

**Comment by v-prasadboke** [comment]
Since we have not received a response in the last 7 days, we are closing your PR #9638 per our standard operating procedures. If you still need support for this issue, you can re-open the PR at any time. 
If you do re-open, we simply request that you ensure the PR has response to the last request. Thank you for your cooperation.

**Pull request closed without merging**